joomla/joomla-cms
# [com_joomlaupdate] Make updating from 3.5.1 to 3.6.x possible without steps between

Pull Request for Update Issues since 3.6.2
### Summary of Changes

Based on the work by @andrepereiradasilva 
With this changes it is possible to update from 3.x direct to 3.6.3
### Testing Instructions
- install 2.5.28
- set this update server `http://www.jah-tz.de/downloads/core/list4.xml`
- Update to 3.5.1 (this step is still needed)
- Update to 3.6.2 with the update provided by the update component
- you need to login to make sure you use this.
- confirm you are on 3.6.2 
- install 3.5.1 (or any other 3.x)
- set the update server `http://www.jah-tz.de/downloads/core/list4.xml`
- Update to 3.6.2
- you need to login to make sure you use this.
- confirm you are on 3.6.2
### Documentation Changes Required

There is no step between 3.\* and 3.6.x anymore
There is a step between 2.5 and 3.6.x (3.5.1)
### Image

Currently it looks like 
![final_login](https://cloud.githubusercontent.com/assets/2596554/17783940/b48d537a-657a-11e6-804b-394266017897.JPG)

I'm happy on any feedback e.g. @wilsonge @mbabker @rdeutz @sinpersister @roland-d on the sec point of the issue as it migrates the sec problem with a login page if that is ok for you guys.

Thanks again to @andrepereiradasilva
